Sources/Charcoal/Filters/Map/MapTabBarController.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+protocol MapTabBarControllerDelegate: AnyObject {
+    func mapTabBarController(_ mapTabBarController: MapTabBarController, didSelect selection: CharcoalViewController.PolygonSelection)
+}
+
+class MapTabBarController: UITabBarController {
```

Comment:
I'm a bit unsure about using a `UITabBarController` here, usually you get issues when it is not in the root of the hierarchy, but maybe it works here because you are not actually using the tab view... Is there any reason for subclassing `UITabBarController` over using a normal `UIViewController` with `children`?

Reply:
There wasn't really a benefit to use `UITabBarController` when I had another look. So now I changed it into a `UIViewController` called `MapFilterViewController`. I also changed so that `MapFilterViewController` subclasses `FilterViewController` (instead of radius and polygon VC doing that), so I handle the bottom button + toggling there, and keep Radius+Polygon as child VC in a `mapContainerView`. You can have another look!